doc/docker/jenkinsnode/Dockerfile

```diff
@@ -0,0 +1,95 @@
+FROM debian:stretch
+
+ARG SWIGVERSION="3.0.11"
+
+# install dependencies
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get -y install \
+  apt-utils \
+  curl \
```

Comment:
Isn't this implied by libcurl4-gnutls-dev

Reply:
it seems to be yes